### PR TITLE
Fix nvtx on Read the Docs

### PR DESCRIPTION
**Pull request by okuta**
`cupy_cuda.h` is broken.

### CODE REVIEW

File: cupy/cuda/cupy_cuda.h [hunk 1]
```diff
@@ -589,17 +589,59 @@ int cudaProfilerStop() {
 // nvToolsExt.h
 ///////////////////////////////////////////////////////////////////////////////
 
+#define NVTX_VERSION 1
```
Comment (gwtnb) [inline cupy/cuda/cupy_cuda.h]: `NVTX_VERSION` of CUDA 8.0RC is 2. Is this value okay?

Comment (okuta) [inline cupy/cuda/cupy_cuda.h]: Version 1 and 2 are both OK.
If use version 1, stub statement is small.

Comment (gwtnb) [inline cupy/cuda/cupy_cuda.h]: I see.

**Comment by gwtnb** [comment]
LGTM

**Pull request merged**